README.md

```diff
@@ -168,6 +168,12 @@ Note: Don't use `@` symbol when adding username.
 When you are done with the setup, you should host your website online.
 We highly recommend to read through the [Deploying on Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) docs for React.
 
+#### Configuring GitHub Actions
+
+- Create a [GitHub CLI Personal Access Token](https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line) with access to public_repo, read:org, workflow, to setup a [GitHub action](https://github.com/JamesIves/github-pages-deploy-action) to deploy to GitHub pages.
```

Comment:
This is not required as they not need to do so. If you needed to link those actions link it at end of readme.

Reply:
removed line